github/Organization.py

```diff
@@ -713,6 +731,22 @@ def has_in_members(self, member):
             )
         return status == 204
 
+    def has_in_pending_members(self, member):
```

Comment:
Since you are referencing this endpoint:
https://developer.github.com/v3/orgs/members/#get-organization-membership

Why not implement a more generic `get_membership` method instead of this?

Reply:
I was following the pattern used in the tests, and discovered that the `has_in_members` method doesn't include members with a state of `pending`. So implementing a similar method for pending members seemed like the better way to go rather than to try to modify `has_in_members` to also handle pending members.

Also, we could implement a more generic `get_membership` method, but the returned Json from the call is different from that returned from the call to `/Users/:user`, meaning we'd either have to return it as just a regular python dict or write a new class to encapsulate the data. The latter is certainly a legitimate option, but I think that's beyond the scope of what this PR is attempting to achieve; we could always open an issue to have that method implemented in a subsequent PR.